language-server/modules/langserver-core/src/test/resources/codeaction/getter-setter/source/getterSetter6.bal

```diff
@@ -0,0 +1,6 @@
+public class Counter {
+
+    private int 'count;
+    private string empName;
+
```

Comment:
If we add the below code segment and what would be the suggested code actions for `'count`?
It should suggest a different getter signature.
```
public function getCount() returns int {
    return 5;
}
```

Reply:
Here in this case, we already have a getter for `'count` hence we will only suggest the setter.